runners/lock_helper.py

```diff
@@ -27,8 +26,8 @@
 
 class Lock(object):
 
-    def __init__(self, path, state_listener=None):
-        client = get_zookeeper_client()
+    def __init__(self, path, state_listener=None, zookeeper_hosts=None):
```

Comment:
What do you think about accepting either a list of strings that we can comma-join, or an already formatted comma-separated string of hosts? That would cover most common cases I believe.